### PR TITLE
Added test results for cirrus-nexus-true-carbon-for-pact-01 from NRI-CTS

### PR DESCRIPTION
**Pull request by levi-ratnakar**

### CODE REVIEW

File: catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "tested_solution": {
+    "solution_id": "cirrus-nexus-true-carbon-for-pact-01",
+    "version": "2.0.0"
+  },
+  "tested_by": {
+    "solution_name": "NRI-CTS",
+    "version": "0.9.15",
+    "solution_id": " nri-cts"
```
Comment (raimundo-henriques) [inline catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json:9]: There is a small typo: a space before `nri-cts`
```suggestion
    "solution_id": "nri-cts"
```
Thank you!

**Review by raimundo-henriques — CHANGES_REQUESTED** [review]
Thank you for the submission!
Please see the inline comment, as there is a small typo there.
Thank you!

File: catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json (hunk 1 above)
Comment (levi-ratnakar) [reply catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json]: @raimundo-henriques Thanks for pointing out the issue. I have fixed and pushed the latest changes. Please review and let us know

Comment (raimundo-henriques) [reply catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json]: Thank you for the changes (and sorry for the delay)! 
It is looking good and we should be able to merge it soon!

**Pull request merged**